webapp/exercise_admin/urls.py

```diff
@@ -10,6 +10,8 @@
         name='file_upload_add'),
     url(r'^file-upload/(?P<exercise_id>\d+)/change$', views.file_upload_exercise,
         {'action': 'change'}, name='file_upload_change'),
+    url(r'^feedback-questions/add$', views.create_feedback_question,
```

Comment:
Move this below the current line 17, so that the url patterns are grouped by the type of models they touch.